studybuddies/src/app/app.module.ts

```diff
@@ -0,0 +1,43 @@
+import { RouterModule, Routes } from '@angular/router';
+
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppComponent } from './app.component';
+import { StudentComponent } from './student/student.component';
+import { TutorComponent } from './tutor/tutor.component';
+import { ClassListStudentComponent } from './student/class-list-student/class-list-student.component';
+import { CreateClassStudentComponent } from './student/create-class-student/create-class-student.component';
+import { ClassListTutorComponent } from './tutor/class-list-tutor/class-list-tutor.component';
+import { CreateClassTutorComponent } from './tutor/create-class-tutor/create-class-tutor.component';
+import { HomeComponent } from './home/home.component';
+
+const routes: Routes = [
+  { path: 'student', component: StudentComponent },
+  { path: 'tutor', component: TutorComponent },
+  { path: 'student/createClass', component: CreateClassStudentComponent },
+  { path: 'student/classList', component: ClassListStudentComponent },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155399)
```suggestion
  { path: "student/classList", component: ClassListStudentComponent },
```